docs/snippets/config/trt-llm-params.mdx

```diff
@@ -0,0 +1,148 @@
+### `trt_llm`
+
+Configuration for TRT-LLM accelerated model services. Here's a TinyLlama example to get started quickly:
+
+```yaml
+model_name: TinyLlama-1.1B-Chat TRT-LLM
+resources:
+  accelerator: A10G
+  use_gpu: True
+trt_llm:
+  build:
+    max_input_len: 1000
+    max_batch_size: 1
+    max_beam_width: 1
+    max_output_len: 500
+    strongly_typed: True
```

Comment:
maybe omit strongly_typed, or are you specifically trying to list all options?

Reply:
want to make this tester fast – strongly typed makes the build faster